beekeeper-package/beekeeper-assembly-metadata-cleanup/pom.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <artifactId>beekeeper-package</artifactId>
+    <groupId>com.expediagroup</groupId>
+    <version>3.0.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>beekeeper-assembly-metadata-cleanup</artifactId>
+  <packaging>pom</packaging>
+
+  <properties>
+    <docker.port>8080</docker.port>
+    <dockerfile-maven-plugin.version>1.4.10</dockerfile-maven-plugin.version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.expediagroup</groupId>
+      <artifactId>beekeeper-metadata-cleanup</artifactId>
+      <version>${project.version}</version>
+      <classifier>spring-boot</classifier>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <finalName>beekeeper-metadata-cleanup-${project.version}</finalName>
```

Comment:
OK, looks like we've gone with `beekeeper-metadata-cleanup*` as the naming throughout so the package name above is fine as it's consistent with that.